Mathlib/Topology/Category/CompHaus/ExplicitLimits.lean

```diff
@@ -109,6 +109,53 @@ def pullback.isLimit : Limits.IsLimit (pullback.cone f g) :=
     (fun _ => pullback.lift_snd _ _ _ _ _)
     (fun _ _ hm => pullback.hom_ext _ _ _ _ (hm .left) (hm .right))
 
+section Isos
+
+noncomputable
+def ToExplicit : Limits.pullback f g ⟶ CompHaus.pullback f g :=
+  pullback.lift f g Limits.pullback.fst Limits.pullback.snd Limits.pullback.condition
+
+noncomputable
+def FromExplicit : CompHaus.pullback f g ⟶ Limits.pullback f g :=
+  Limits.pullback.lift (pullback.fst _ _) (pullback.snd _ _) (pullback.condition f g)
+
+@[simp]
+theorem toExplicit_comp_fromExplicit_eq_id :
+    (ToExplicit f g ≫ FromExplicit f g) = 𝟙 _ := by
+  refine' Limits.pullback.hom_ext (k := (ToExplicit f g ≫ FromExplicit f g)) _ _
+  · simp [ToExplicit, FromExplicit]
+  · rw [Category.id_comp, Category.assoc, FromExplicit, Limits.pullback.lift_snd,
+      ToExplicit, pullback.lift_snd]
+
+@[simp]
+theorem fromExplicit_comp_toExplicit_eq_id :
+    (FromExplicit f g ≫ ToExplicit f g) = 𝟙 _ :=
+  pullback.hom_ext f g _ _ (by simp [ToExplicit, FromExplicit]) (by simp [ToExplicit, FromExplicit])
+
+@[simps]
+noncomputable
+def FromExplicitIso : CompHaus.pullback f g ≅ Limits.pullback f g where
```

Comment:
Is all the stuff above really necessary, or can you just define this iso using `pullback.isLimit` and `conePointUniqueUpToIso` and work with that instead? Similarly for finite coproducts.

Reply:
You're right, `conePointUniqueUpToIso worked